packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -156,9 +169,15 @@ if (__DEV__) {
       JSON.stringify(normalizedServerValue),
       JSON.stringify(normalizedClientValue),
     );
+    if (!isJsdomDev) {
```

Comment:
They're pretty noisy in Node environment because it starts printing fibers due to the reference inside. Plus not very helpful anyway. Not sure if there's a better way to detect.